web/styles/zulip.css

```diff
@@ -2035,6 +2041,22 @@ body:not(.hide-left-sidebar) {
         .app-main .column-middle {
             margin-left: 7px;
         }
+
+        #realm-navbar-icon-logo {
+            display: inline-block;
+        }
+
+        #realm-logo {
```

Comment:
Might want to rename this to `realm-navbar-wide-logo`, and maybe use `realm-navbar-square-logo`, for better contrast?